distributed/cli/utils.py

```diff
@@ -69,3 +71,12 @@ def uri_from_host_port(host_arg, port_arg, default_port):
     addr = unparse_address(scheme, loc)
 
     return addr
+
+
+def create_ssl_context(certfile, keyfile):
+    if certfile and keyfile:
```

Comment:
This doesn't seem right. Theoretically, you can create a context with only a certfile and no keyfile (for example if the certfile also contains the private key).

If that's not allowed, then passing only one of two options should give an error instead of silently falling back on an insecure transport.